ui/widgets/resizable.js

```diff
@@ -226,6 +226,9 @@ $.widget( "ui.resizable", $.ui.mouse, {
 			this._removeHandles();
 			this._setupHandles();
 			break;
+		case "aspectRatio":
+			this._aspectRatio = !!value;
```

Comment:
Add tests for setting `aspectRatio` numeric value. Commit https://github.com/jquery/jquery-ui/pull/1750/commits/02796d6dac21c088f30003c11c331cf3499f67ca